### PR TITLE
Design app sidebar

### PR DESCRIPTION
**Pull request by neurosnap**
# BEFORE
Screen Shot 2023-02-10 at 9 54 54 AM

# AFTER
Screen Shot 2023-02-10 at 9 53 58 AM

### CODE REVIEW

File: index.css [hunk 1]
```diff
@@ -3,6 +3,7 @@
 @import "tailwindcss/utilities";
 
 html, body, #app {
+  font-size: 14px;
```
Comment (madhuravius) [inline index.css:6]: i'm accustomed to using em/rem and not hardcoding this (using scss vars)

maybe TODO/discussion for later, not actionable now

File: src/app/router.tsx [hunk 2]
```diff
@@ -147,12 +148,17 @@ const appRoutes: RouteObject[] = [
...
-        path: routes.TEAM_PATH,
-        element: <TeamPage />,
+        element: <SettingsPageLayout />,
+        children: [
+          {
+            index: true,
+            element: <SettingsPage />,
+          },
+          {
+            path: routes.TEAM_PATH,
```
Comment (madhuravius) [inline src/app/router.tsx:158]: intended to use this layout?

Comment (neurosnap) [reply src/app/router.tsx:158]: Yep!

File: index.css (hunk 1 above)
Comment (neurosnap) [reply index.css:6]: `rem` means: relative to the root `font-size`.  So we have to set an explicit font-size in pixels or else it defaults to 16px.

**Review by madhuravius — APPROVED** [review]

**Pull request merged**